index.php

```diff
@@ -153,6 +153,12 @@
                         <img src="images/person-search.svg">
                         <span>Find Family Account</span>
                 </div>
+
```

Comment:
This portion of the code needs to be removed. This button is on the staff dashboard, and this does not apply to the staff. When a staff views a family profile after searching for one, it still pulls up the newer format which is good though.

Reply:
oops sorry i was trying to test it when I couldn't log into family account, I'll removed it